content/docs/card/feature-card-tsx.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Feature Card Tsx
+description: pill-shaped button designs.
```

Comment:
Improve title and description.

title: Integration pills
description: A component that displays a list of services it can integrate with.

Reply:
okay brother , i will